wheel/build_wheel.sh

```diff
@@ -134,6 +134,8 @@ export TH_BINARY_BUILD=1
 export INSTALL_TEST=0 # dont install test binaries into site-packages
 export MACOSX_DEPLOYMENT_TARGET=10.10
 export CMAKE_PREFIX_PATH=${CONDA_PREFIX:-"$(dirname $(which conda))/../"}
+export USE_LLVM="/opt/llvm"
```

Comment:
These scripts are mainly used for mac wheels which probably won't use this feature?

Reply:
I think using this for mac is reasonable, llvm is pretty well supported there, the CPU backend is probably useful there, and the nnc team has the ability to test it locally.  So I was going to try to land it there unless it really complicates things.

Reply:
Per @malfet I can leave this off for Mac so we get coverage of a non-llvm enabled build.  I think the Linux builds are the ones I most want to have this anyways.